app/App.js

```diff
@@ -1,3 +1,5 @@
+// @flow
+import { hot, setConfig } from 'react-hot-loader';
```

Comment:
move hot reload to index.js where it was initially

Reply:
ok